app/controllers/api/v1/projects_controller.rb

```diff
@@ -1,11 +1,10 @@
 module Api
   module V1
     class ProjectsController < Api::ApplicationController
+      before_action :set_project, only: %i[update destroy]
       def index
         @projects = Project.includes(:tasks)
                            .order(params[:sort])
-                           .page(params[:page])
-                           .per(3)
```

Comment:
For what?

Reply:
@Timchik-IT

Reply:
there I just copied from the controller, but this is the same display in insomnia, so I removed it

Reply:
Please leave it, you can use it with request params